main/alpine-baselayout/APKBUILD

```diff
@@ -195,6 +195,8 @@ package() {
 			sed -i "s/tty$i::/\#tty$i::/g" "$srcdir"/inittab
 		done
 		echo "console::respawn:/sbin/getty 38400 /dev/console" >> "$srcdir"/inittab
+	elif [ "$CARCH" = "ppc64le" ]; then
+		echo "console::respawn:/sbin/getty -L /dev/console 0 vt100" >> "$srcdir"/inittab
```

Comment:
I wonder if we should do this in initramfs init instead. There are other systems that use hvc0, xen for example.

See https://github.com/alpinelinux/mkinitfs/blob/master/initramfs-init.in#L108

Reply:
I have only tested ppc64le but would other architectures be ok with `console::respawn:/sbin/getty -L /dev/console 0 vt100`? Or would there need to be an `if ppc64le || xen`?